build/signing-config-mac.yaml

```
@@ -0,0 +1,48 @@
+# This file describes the code signing configuration for macOS.
+
+# List of entitlements.
+entitlements:
+  # This contains the default entitlements, for files not otherwise listed.
+  default:
+  - com.apple.security.inherit
+
+  # Entitlement overrides.  This is a list of overrides, each with a "paths"
+  # key describing which paths to override, and an "entitlements" key for the
+  # overriding entitlements.
+  overrides:
+  - paths:
+    - '' # This is the main application
+    entitlements:
+    - com.apple.security.cs.allow-jit
+  - paths:
+    - Contents/Resources/resources/darwin/lima/bin/limactl
+    - Contents/Resources/resources/darwin/lima/bin/limactl.ventura
+    - Contents/Resources/resources/darwin/lima/bin/qemu-system-aarch64
```

Not sure if you can add entries that may not exist, but we need to include qemu on amd64 as well:

```suggestion
    - Contents/Resources/resources/darwin/lima/bin/qemu-system-aarch64
    - Contents/Resources/resources/darwin/lima/bin/qemu-system-x86_64
```